clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -446,8 +446,31 @@ OpFoldResult cir::CastOp::fold(FoldAdaptor adaptor) {
 // CallOp
 //===----------------------------------------------------------------------===//
 
+mlir::Operation::operand_iterator cir::CallOp::arg_operand_begin() {
+  assert(!cir::MissingFeatures::opCallIndirect());
```

Comment:
Will we issue a diagnostic somewhere for indirect calls? This seems like a significant gap.

Reply:
This function has been removed in the latest change.